### PR TITLE
[v1.5][W14-B4] CelebManager

### PR DESCRIPTION
**Pull request by muruges95**
Start of pull request. Added documentation on user stories, use cases, NFR etc. in our Developer Guide for the Week 7 requirements. Will update with our individual enhancements to the different components of the address-book. The name of the addressbook will be Celeb Scheduler, as mentioned in the title. Unclear if we are supposed to put this name in the Developer Guide as well, but for now that has not been done.

Other team members (so they get notified): @Damienskt @WJY-norainu @tzerbin

### CODE REVIEW

**Comment by muruges95** [comment]
As of now this is our v1.0 of our app.

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level4
+:repoURL: https://github.com/CS2103W14-B4/addressbook-level4
```
Comment (kowshik-sundararajan) [inline docs/UserGuide.adoc]: Remember to change the repo name to `main`

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 2]
```diff
@@ -67,4 +69,24 @@ public boolean equals(Object other) {
...
+            labelTags.getStyleClass().add(getColorForTags(tag.tagName));
+            tags.getChildren().add(labelTags);
+        });
+    }
+    /**
+    * Returns color style for {@code tagName}'s label.
+    */
+    private String getColorForTags(String tagName) {
+        /**
+         * we use the hash code of the tag name to generate a random color, so that the color remain consistent
```
Comment (kowshik-sundararajan) [inline src/main/java/seedu/address/ui/PersonCard.java]: Could you rephrase this comment to describe what the code does and not what the developer does?

File: README.adoc [hunk 3]
```diff
@@ -1,11 +1,9 @@
 = Address Book (Level 4)
```
Comment (kowshik-sundararajan) [inline README.adoc]: Remember to update the name of your app - it is not an Address Book anymore!

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -10,7 +10,7 @@ ifdef::env-github[]
 :tip-caption: :bulb:
 :note-caption: :information_source:
 endif::[]
-:repoURL: https://github.com/se-edu/addressbook-level4/tree/master
+:repoURL: https://github.com/CS2103W14-B4/addressbook-level4/tree/master
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
```
Comment (kowshik-sundararajan) [inline docs/DeveloperGuide.adoc]: Team name should be updated as well.

**Review by kowshik-sundararajan — COMMENTED** [review]
Good work on updating user stories, use cases and other NFRs 👍 

Please check Travis to check why the build is failing and make the necessary changes to ensure that the build passes. 

Some general feedback:
* Good to see that you have created issues and `v1.0` milestone. Please remember to add `v1.1` to `v1.5` milestones by next tutorial - they can be empty but should be created with deadlines.

* Remember to update the target user profile and the description of the app.

* Do set up the issue tracker in your repo. 

* I see that individual PRs that didn't pass Travis have been merged in your repo - in practice, this shouldn't be acceptable. By merging pull requests that do not pass Travis, you will find it difficult to figure out why the main build fails.

*Do not close this PR!*

**Comment by muruges95** [comment]
Can I just check, do we rename the pull request to v1.1 for this coming  week?

**Comment by muruges95** [comment]
The reason why the build is failing is because of a style issue on upstream's version of the code; specifically in the file XMLAddressBookStorage.java .Locally we dont have any travis build fails, if you go to our repo page, you can see that both the AppVeyor and Travis builds are fine. 

image
The above is an image from the travis for the check of this pull request. I notice that no other team is facing this issue, so would like to know what we did wrong and how to resolve this.

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -844,16 +945,461 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+=== Use case: Auto-fill
+
+*MSS*
```
Comment (kowshik-sundararajan) [inline docs/DeveloperGuide.adoc]: Remember to update all references to `AddressBook` to your application's name!

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -11,13 +11,13 @@ ifdef::env-github[]
...
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
```
Comment (kowshik-sundararajan) [inline docs/UserGuide.adoc]: Remeber to update the `By` (similar to Dev Guide).

**Review by kowshik-sundararajan — COMMENTED** [review]
Good job on updating the documentation.

@muruges95  @WJY-norainu  @Damienskt  @tzerbin 
Please indicate which features appear in which milestone. Also If there is something that is left for future work, need to indicate it as coming in v2.0.

I would want you to mention the following in your **Developer Guide**:

* For each team member what are the features (major and minor) you are proposing?
* Within the scope of the project, how does it fit in (just a one or 2 line summary)
Please do this by end of day Monday 19 March.

General feedback: 
* Include features and their description that are planned to be implemented in `v2.0` in the User Guide.
* Remember to replace all references to AddressBook 😛 
* You can be more specific in your user stories / dev guide - replacing person with celebrity. Ex: Add a celebrity, etc. 
* If you want me to review your code for quality checks, tag me in the PRs submitted to your repo.

*Note for next week*: Each member of the team should have at least 1 merged PR of their enhancement as `v1.2` release. (It doesn't have to be perfect code, but something that works).

As a favour, could you add a comment every weekend to summarize what are the changes that have been made? This would make it easier for me to review the changes.

File: docs/UserGuide.adoc (hunk 6 above)
Comment (muruges95) [reply docs/UserGuide.adoc]: Noted

File: docs/DeveloperGuide.adoc (hunk 5 above)
Comment (muruges95) [reply docs/DeveloperGuide.adoc]: Noted

**Review by kowshik-sundararajan — COMMENTED** [review]
@muruges95 @WJY-norainu @Damienskt @tzerbin 
Since I've reviewed the DG and your enhancements in the tutorial, there's not much to add on here.

For your UG, good that you have mentioned which features were implemented in `v1.2` and which features will be implemented in `v1.3`. You can also specify which features will be implemented for the remaining milestones. (Basically, all the features should be present and they should have a milestone specified for each of them).
